test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorProviderTest.cs

```diff
@@ -37,111 +37,6 @@ public void GetValidators_ReturnsValidatorForIValidatableObject()
 #endif
 
         [Fact]
-        public void GetValidators_DoesNotAddRequiredAttribute_ForNonNullableValueTypes_IfAttributeIsSpecifiedExplicitly()
```

Comment:
where are the replacements for the tests removed from this class?  don't see a `DataAnnotationsClientModelValidatorProviderTest` class.